[ONNX] Add comment on test_view_dynamic_zero_dim

titaiwangms opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #105951
* __->__ #105950

From https://github.com/pytorch/pytorch/issues/105066. 

The case was meant to test 0 bbox generated by vision models, but the bboxes still have `.view()` operated. The case was disabled, and not supported. Added a comment to clear the potential confusion in the future. We will wait for model example to proceed on this.

titaiwangms committed:
[ONNX] Remove test_view_dynamic_zero_dim

[ghstack-poisoned]

BowenBao commented on test/onnx/test_fx_to_onnx_with_onnxruntime.py:
```diff
@@ -566,21 +566,21 @@ def forward(self, x):
         self.run_test_with_fx_to_onnx_exporter_and_onnx_runtime(
             ViewModel(),
             (x,),
+            # additional_test_inputs=[((y,),)],
```
Is this actually being tested in current CI? I thought it was skipped by a change.

BowenBao replied:
ah this is plain new code, so it was already removed lol.

thiagocrepaldi reviewed (approved):
LGTM. Add a TODO to uncomment the addition `y` input soon?

titaiwangms commented:
@pytorchbot merge

titaiwangms commented:
@pytorchbot merge

titaiwangms committed:
Update on "[ONNX] Add comment on test_view_dynamic_zero_dim"


From https://github.com/pytorch/pytorch/issues/105066. 

The case was meant to test 0 bbox generated by vision models, but the bboxes still have `.view()` operated. The case was disabled, and not supported. Added a comment to clear the potential confusion in the future. We will wait for model example to proceed on this.

[ghstack-poisoned]
